tests/ganache.go

```diff
@@ -165,7 +165,7 @@ func getPackageKeys(t *testing.T, packageName string) []string {
 		startIndex += pkg.numKeys
 	}
 	t.Fatalf("Package %q does not have reserved test keys", packageName)
-	panic("unreachable code")
+	return nil // unreachable
```

Comment:
Newer linter complained that the panic was unreachable, but was okay with returning `nil`.